### ISSUE TITLE
Unable to download S3 files

### ISSUE THREAD

**Issue by rpiskule** [awslabs/aws-ant-tasks#15]
Exception while trying to download object: AWS authentication requires a valid Date or x-amz-date header

**Comment by rtnewton1** [comment]
Exactly. I can't run an upload task either. It seems that AWS has changed there API, but this library had not kept up with it.

**Comment by fulghum** [comment]
Sorry for the slow response on this one.  Do you guys happen to have any HTTP wire logs from this error?  The S3 API has remained backwards compatible since it's original release in 2006, so older SDK versions should work just fine.  I'll see if we can repro the issue over on our side and find some more clues.

**Comment by GenghisChen** [comment]
I encountered the same problem and found clues in this thread: https://github.com/aws/aws-sdk-java/issues/484
The problem is related to new Java releases and joda-time releases prior to 2.8.1

To solve it, i replaced the content of the _/org/joda/time_ folder inside aws-java-sdk-ant-tasks-1.2.3.jar with the content of the latest joda-time.jar (2.9.4)

**Comment by rtnewton1** [comment]
Outstanging!

**Comment by terraframe** [comment]
The fix mentioned in this ticket can be downloaded from:

http://nexus.terraframe.com/service/local/repositories/thirdparty/content/com/amazonaws/aws-java-sdk-ant-tasks/1.2.3-2/aws-java-sdk-ant-tasks-1.2.3-2.jar

**Comment by tylervz** [comment]
@terraframe thank you. Using that jar worked for me.
